fmriprep/interfaces/images.py

```diff
@@ -279,6 +228,93 @@ def _run_interface(self, runtime):
         return runtime
 
 
+class ConformSeriesInputSpec(BaseInterfaceInputSpec):
+    t1w = File(exists=True, mandatory=True,
```

Comment:
The reasons for using `t1w_list` are now obviated, so I'd switch to `in_file`, to make this more generic.